flink-table/flink-sql-client/pom.xml

```diff
@@ -109,6 +109,13 @@ under the License.
 			<version>3.9.0</version>
 		</dependency>
 
+		<!-- Tools to Unify display format for different languages -->
+		<dependency>
+			<groupId>com.ibm.icu</groupId>
```

Comment:
We should also shade this jar into `flink-sql-client` jar?
If yes, then please also update the license files